src/components/NewsletterSignup/NewsletterSignup.tsx

```diff
@@ -0,0 +1,193 @@
+import {
+  Box,
+  chakra,
+  Stack,
+  useColorModeValue,
+  useStyleConfig,
+  VStack,
+} from "@chakra-ui/react";
+import React, { forwardRef } from "react";
+
+import Button from "../Button/Button";
+import Form, { FormField } from "../Form/Form";
+import Icon from "../Icons/Icon";
+import Link from "../Link/Link";
+import Text from "../Text/Text";
+import TextInput from "../TextInput/TextInput";
+import Heading from "../Heading/Heading";
+import useNYPLBreakpoints from "../../hooks/useNYPLBreakpoints";
+import { getSectionColors } from "../../helpers/getSectionColors";
+import { SectionTypes } from "../../helpers/types";
+
+interface NewsletterSignupProps {
+  /** Optional: Additional class name to add. */
```

Comment:
I noticed that this pattern of prefixing each prop description with "Optional" or "Required" is only on this component present. I feel like it might be better to keep it in line with the other storybook entries but I might be missing something?

Reply:
Sure, you can remove it. It's also implied with the `?` in the prop definition.